schemas/TransferTemplate.json

```diff
@@ -65,6 +61,21 @@
       "minItems": 1,
       "uniqueItems": true,
       "items": { "$ref": "FundsTemplate.json" }
+    },
+    "additional_info": {
+      "description": "Includes part_of_payment, case, etc.",
+      "type": "object",
+      "properties": {
+        "part_of_payment": {
```

Comment:
This is a slight leakage through the abstraction since the ledger isn't supposed to be aware of the concept of a payment. It might be preferable to call it something generic like "tag" to avoid this issue.